go.mod

```diff
@@ -44,4 +44,4 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-replace github.com/nedpals/supabase-go => github.com/jan-zabloudil/supabase-go v0.5.0
+replace github.com/nedpals/supabase-go => github.com/jan-zabloudil/supabase-go v0.6.0
```

Comment:
A new tag has been published with fixes for generating signed URLs

https://github.com/jan-zabloudil/supabase-go/releases/tag/v0.6.0